_dev/src/utils/LocalStorage.ts

```diff
@@ -15,4 +15,12 @@ export function getDataFromLocalStorage(key: string) {
   }
 }
 
+export function deleteDataFromLocalStorage() {
```

Comment:
```suggestion
export function deleteProductFeedDataFromLocalStorage() {
```

So we can have other methods later that would deal with other data.